CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 ## v0.5.5 (Unreleased)
 
+### Notes
+
+- RNS resolution is now available in addition to the existing ENS resolution
```

Comment:
Would it be helpful to explain what RNS is or to link to https://developers.rsk.co/rif/rns/?

Reply:
sure. thanks!